lightningd/lightningd.c

```diff
@@ -251,6 +254,31 @@ static void daemonize_but_keep_dir(void)
 	tal_free(cwd);
 }
 
+static void pidfile_create(const struct lightningd *ld)
+{
+        int pid_fd;
+        char *pid;
+        const tal_t *tmpctx = tal_tmpctx(NULL);
+
+        /* Create PID file */
+        pid_fd = open(ld->pidfile, O_WRONLY|O_CREAT, 0640);
+        if (pid_fd < 0)
+                err(1, "Failed to open PID file: %s", strerror(errno));
```

Comment:
Oops, sorry, I wasn't clear. "err()" already appends the ": %s", strerror(errno)" part, so you can delete this :)